app/services/users/suggest_recent.rb

```diff
@@ -58,13 +58,13 @@ def relation_as_array(relation, limit:)
 
     def established_user_article_count
       Rails.cache.fetch("established_user_article_count", expires_in: 1.day) do
-        User.where("articles_count > 0").average(:articles_count) || User.average(:articles_count)
+        User.where(articles_count: 1..).average(:articles_count) || User.average(:articles_count)
```

Comment:
I noticed this still used the older AR syntax, so since I was here already I decided to change this too.